src/server/pmix_server_get.c

```diff
@@ -199,7 +199,14 @@ pmix_status_t pmix_server_get(pmix_buffer_t *buf,
          * give the host server a chance to tell us about it */
         rc = create_local_tracker(nspace, rank, info, ninfo,
                                   cbfunc, cbdata, &lcd);
-        return rc;
+/*
+ * Its possible there are no local processes on this host, so lets ask 
+ * for this explicitly.   
+ */
+        if( NULL != pmix_host_server.direct_modex ){
+                pmix_host_server.direct_modex(&lcd->proc, info, ninfo, dmdx_cbfunc, lcd);
```

Comment:
Actually, I'm not sure we need to check for trackers either.  Won't that already happen once the last local rank registers itself?  At that point we call pmix_pending_nspace_requests(nptr) when the last _register_client call happens?